avaje-config/src/main/java/io/avaje/config/InitialLoader.java

```diff
@@ -272,6 +285,23 @@ boolean loadWithExtensionCheck(String fileName) {
     }
   }
 
+  private boolean loadURI(String uriPath) {
+    var uri = URI.create(uriPath);
+    final var scheme = uri.getScheme();
```

Comment:
FYI `getScheme()` is nullable.

I'm not sure I like calling uriLoaders.get(`null`). I realize it works for `HashMap` but you will get a NPE if it is `ConcurrentHashMap`.